### DISCUSSION TITLE
Is the Data transcribed from the audio file safe?

### DISCUSSION THREAD

**Discussion by muizzhaque** [NVIDIA-NeMo/NeMo#6966] [Q&A] [answered]
I’m planning to use Nvidia Nemo framework for speech diarization, speech recognition and other ASR operations and right now I’m doubtful about the data privacy coz the data will have confidential information and i don’t want it to be leaked in any way. can you walk me through the entire process of encrytion and data’s lifecycle. Also, will this be stored in your cloud.

**Comment by titu1994** [comment, marked as answer]
There is no encryption in Nemo. All code runs locally so user has to manage data storage and writing of out result to manifest file is also on local storage.
